joomla/joomla-cms
# [4.0] Refactored admin-item-edit_container.es6.js to no longer use jQuery anymore

### Summary of Changes
Removed any jQuery code and rebuilt it with ECMAScript 6.


### Testing Instructions


1. Login into the backend of your joomla! site

2. (Toggle the side navigation if hidden)

3. Click on "Menus" and choose an administrator menu if available. If not, you'll need to create one

4. After being redirected to the "Menus: Items" page: Click on "new" to create an new item

5. Now you should be able to select the menu item type. Click on "select"

6. Choose: "System Links". A dropdown will open. There, you have to choose: "Components Menu Container"

7. At the middle of the page should be a list in form of a tree.

### Expected result
The list has sevaral "root" elements, which is, a list-item, which contains a subtree. 
Current "root elements" on this page are: "All", "Banners", "Contacts", "Messaging" and "News Feeds".
Clicking on one of them should instantly check and disable all elements, which are hold inside of this root.

For example: Clicking on "All" should check and disable every checkbox on the screen. Of course, if you click it again, every checkbox will be unchecked and enabled. 

* Check means, that the checkbox color and text will change to red
* disable means, that the checkbox is still red, but lighter due to less opacity

If a checkbox is either checked nor disabled it's default color is green.

Clicking on a item, which is not a root, will be checked(color switches to red) but not disabled, because you wouldn't be able to click it again. 

Now apply the patch  from this PR , re-run npm and verify that the functionality described above still works as expected.
